lib/cobrato/resources/charging_type.rb

```diff
@@ -0,0 +1,21 @@
+module Cobrato
+  module Resources
+    class ChargingType < Base
+
+      def collection_name
+        'charging_types'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.